unittests/abi_tests.cpp

```diff
@@ -530,6 +530,44 @@ fc::variant verify_type_round_trip_conversion( const abi_serializer& abis, const
 }
 )=====";
 
+BOOST_AUTO_TEST_CASE(std_array_types)
+{ try {
+
+   const char* currency_abi = R"=====(
```

Comment:
Rename to `test_abi`

Reply:
I don't really mind, but this doesn't seem consistent with the other tests in the same file.

Reply:
I mean `currency_abi` => `test_abi`

Reply:
Got it, makes sense, thanks!